webapp/i18n/en.json

```diff
@@ -136,10 +137,20 @@
   "EditableDayPicker.today": "Today",
   "Error.mobileweb": "Mobile web support is currently in early beta. Not all functionality may be present.",
   "Error.websocket-closed": "Websocket connection closed, connection interrupted. If this persists, check your server or web proxy configuration.",
+  "Filter.contains": "contains",
```

Comment:
Wow.  It looks like 118n extraction hadn't been run in a while.  We should add that to our CI/lint.

Reply:
I'll create a ticket for that.